x-pack/plugins/endpoint/common/schema/alert_index.ts

```diff
@@ -44,10 +43,10 @@ export const alertingIndexGetQuerySchema = schema.object(
       schema.string({
         validate(value) {
           try {
-            fromKueryExpression(value);
+            decode(value);
```

Comment:
Shouldn't this be decoded and then run through `fromKueryExpression` still? To check both cases? Or are we changing the format of `query`?

Reply:
@madirey We're changing the format of query to be of type `Query`, which has a `query` and a `language` (kql or lucene)